### PR TITLE
add authorize edition constraint

### PR DESCRIPTION
**Pull request by nicolasThal**

### CODE REVIEW

**Comment by nicolasThal** [comment]
https://github.com/open-orchestra/open-orchestra-cms-bundle/pull/1018
https://github.com/open-orchestra/open-orchestra-model-interface/pull/107
https://github.com/open-orchestra/open-orchestra-libs/pull/10
https://github.com/open-orchestra/open-orchestra-model-bundle/pull/405

File: BackofficeBundle/Validator/Constraints/AuthorizeEditionValidator.php [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+        $this->authorizeEditionManager = $authorizeEditionManager;
+    }
+
+    /**
+     * Checks if the passed value is valid.
+     *
+     * @param mixed $value The value that should be validated
+     * @param Constraint $constraint The constraint for the validation
+     *
+     * @api
```
Comment (itkg-alavieille) [inline BackofficeBundle/Validator/Constraints/AuthorizeEditionValidator.php]: why this annotation ?

**Comment by nicolasThal** [comment]
@itkg-alavieille pr updated

**Comment by itkg-alavieille** [comment]
:+1:

**Pull request merged**